hydromt_sfincs/sfincs.py

```diff
@@ -2851,6 +2902,17 @@ def read_grid(self, data_vars: Union[List, str] = None) -> None:
                 ds.raster.set_crs(epsg)
             self.set_grid(ds)
 
+            # TODO - fix this properly; but to create overlays in GUIs,
```

Comment:
This is only used by FloodAdapt (and/or DelftDashboard) for the grid-snapping (and eventually visualization of the grid). 
@LuukBlom should we make this optional maybe, that would require a change on the FA side.

Reply:
In FA we access the grid of SfincsModels only via the SfincsModel.quadtree attribute, but we can change that easily to some other function / property, so changes for FloodAdapt should be easy. 
We also cache the result of this grid attr/function on our side, so that reduces worries about performance from us.

If you want to put this in a function somewhere to make it optional, but still allow us to retrieve the grid, Im all for it! Please do let me know which function:)